misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java

```diff
@@ -46,6 +46,13 @@ public class Messages {
 
     private static final String NO_BALLERINA_FILE_OR_MODULE = "couldn't find an existing ballerina file or " +
             "module by the name: ";
+
+    private static final String CANT_ALLOW_BOTH_MODULE_AND_FILE_OPTIONS =
+            "too many options: either use module option or file option";
```

Comment:
```suggestion
            "using '--module' and '--file' options at the same time for selective formatting is not allowed";
```

Reply:
@praneesha WDYT?

Reply:
How about `Use one of the '--module' or '--file' options for selective formatting."

Reply:
yes, +1